dataquality/schemas/seq2seq.py

```diff
@@ -41,9 +47,11 @@ def set_target(cls, df: DataFrame) -> DataFrame:
 
 class Seq2SeqOutputCols(str, Enum):
     id = "id"
-    dep = "data_error_potential"
-    token_deps = "token_deps"
-    token_gold_probs = "token_gold_probs"
+    dep = "data_error_potential"  # TODO Depricate
+    perplexity = "perplexity"  # TODO Depricate
+    token_deps = "token_deps"  # TODO Depricate
```

Comment:
```suggestion
    perplexity = "perplexity"  # TODO Depricate
```

nice. good to deprecate the dep ones

Reply:
Wait keep perplexity for now?

Reply:
I guess I can keep for now. But we should in the future move to runners. I will create a quick ticket.

Reply:
https://app.shortcut.com/galileo/story/7593